features/networking/sdn.feature

```diff
@@ -66,53 +66,69 @@ Feature: SDN related networking scenarios
   @admin
   @destructive
   Scenario: SDN will detect the version and plugin type mismatch in openflow and restart node automatically
-    Given the master version >= "3.7"
-    And I select a random node's host
-    Given the cluster network plugin type and version and stored in the clipboard
-    And system verification steps are used:
-    """
-    When I run ovs dump flows commands on the host
-    Then the step should succeed
-    Then the output should contain "<%= cb.net_plugin[:type] %>.<%= cb.net_plugin[:version] %>"
-    """
-    And the node service is verified
-    And the node network is verified
-
-    When I run the ovs commands on the host:
-      | ovs-ofctl -O openflow13 mod-flows br0 "table=253, actions=note:<%= cb.net_plugin[:type] %>.ff" |
-    Then the step should succeed
+    Given the master version >= "3.11"
```

Comment:
@anuragthehatter 
Thanks your effort.
 BTW, that's mean this case would cannot be tested before 3.11 after the fixed?

Reply:
Actually this can be also tested on 3.10 as well so it can be changed as >="3.10". As starting 3.10, sdn ovs services are containerized while pre 3.10 sdn/ovs services were running on the node. So i am not sure at this point if this case also need to be supported on 3.9 and down or depends how many testing requests we get on 3.9 or before to find out if its worth the efforts and time. If that's the case i need to spend good time on figuring out like if version less than 3.10 focus on journalctl logs else sdn pod logs and otehr stuff etc or a separate scenario needs to be created for version less than 3.10 as whole scenario will be changed - no pods, no oc logs but just commands on host and journalctl on node. It seems difficult but not impossible to make this one scenario works for all version. Thanks! I guess this calls for a discussion.

Reply:
Anything that you want to run on 3.x must go as a separate PR to the `v3` branch.. just FYI

Reply:
@akostadinov Please help me understand bit more on it. Say i have a scenario which says "Given master version >=3.7" which means that this scenario can run on 3.x and 4.x both. Where would you add the scenario, in master branch or v3?

Reply:
@anuragthehatter - both

Reply:
@akostadinov make sense, thanks. Gonna have to work on this PR again, currently engaged in high priority stuff